sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -809,15 +808,14 @@ def test_dofn_client_process_performs_batching(self):
     fn.process(('project_id:dataset_id.table_id', {'month': 1}))
 
     # InsertRows not called as batch size is not hit yet
-    self.assertFalse(client.tabledata.InsertAll.called)
+    self.assertFalse(client.insert_rows_json.called)
```

Comment:
Do you know if the API support other formats (for example, Avro) that are more efficient ?

Reply:
no. The API formats everything into JSON. Other API calls use automatic JSON conversions.